harness.yml

```diff
@@ -5,6 +5,9 @@ harness('inviqa/go'):
       - proxy
     confd:
       - harness:/
+    attributes:
+      standard:
+        - module_name
```

Comment:
I've not seen this syntax before

Reply:
it will prompt the user for this value when you do `ws create ...` 👍